### PR TITLE
Add: Brazilian Localization

### PR DESCRIPTION
**Pull request by andercard0**
Translated the application to Brazilian Portuguese.

## Description
App localization to a new language

## Motivation and Context
Feature more language is good.

## How Has This Been Tested?
Not tested. actually I did receive some weird erros while tried to build, maybe due to a lack of workflow on my end.
Tho i don't think it's fully necessary atm due the fact this is just a json adition. If any further test is needed let me
know.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [ X ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/PicView.Core/Config/Languages/pt-br.json [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+  "ColorPickerToolTooltip": "Escolha a cor da imagem",
+  "ColorRepresentation": "Representação de cores",
+  "ColorTone": "Tonalidade da cor",
+  "CompressedBitsPixel": "Bits compactados por pixel",
+  "Compression": "Compressão",
+  "Contrast": "Contraste",
+  "ConvertTo": "Converter para",
+  "ConvertedToBase64": "Convertido para base64",
+  "CoolWhiteFluorescent": "Branco fluorescente frio",
+  "CopiedImage": "Imagem copiada para a área de transferência,
```
Comment (Ruben2776) [inline src/PicView.Core/Config/Languages/pt-br.json]: "CopiedImage": "Imagem copiada para a área de transferência",

File: src/PicView.Core/Config/Languages/pt-br.json [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+  "SourceFolder": "Pasta de origem",
+  "Space": "Espaço",
+  "Square": "Quadrado",
+  "Start": "Iniciar",
+  "StartSlideshow": "Iniciar apresentação",
+  "StayCentered": "Manter a janela centralizada",
+  "StayTopMost": "Manter a janela sobreposta",
+  "Stretch": "Esticar",
+  "StretchImage": "Esticar imagem",
+  "StrobeReturnLightDetected": Detecção de luz de retorno estroboscópica",
```
Comment (Ruben2776) [inline src/PicView.Core/Config/Languages/pt-br.json]: "StrobeReturnLightDetected": "Detecção de luz de retorno estroboscópica",

**Review by Ruben2776 — CHANGES_REQUESTED** [review]

File: src/PicView.Core/Config/Languages/pt-br.json (hunk 1 above)
Comment (andercard0) [reply src/PicView.Core/Config/Languages/pt-br.json]: Fixed

File: src/PicView.Core/Config/Languages/pt-br.json (hunk 2 above)
Comment (andercard0) [reply src/PicView.Core/Config/Languages/pt-br.json]: Fixed

**Comment by Ruben2776** [comment]
Thank you very much!

**Pull request merged**